int32 damage_index
---
trajectory_msgs/MultiDOFJointTrajectory trajectory   # repair trajectory